### PR TITLE
[Club21-15] Implement Referral page UI

### PR DESCRIPTION
**Pull request by trgianghuynh1808**
#15

### CODE REVIEW

**Comment by thaind97git** [comment]
image
- Two cards should behave a little higher and follow with [this position](https://drive.google.com/file/d/1-MLc0EWs3eMkcunxlXMngq303y1BvaRZ/view)
- The table shouldn't have box-shadow and should behave `striped prop`

**Comment by thaind97git** [comment]
image
The header titles not bold.
Content has to margin-left.
The input copy doesn't display full.

**Comment by thaind97git** [comment]
image

Some things not the same

File: components/UniLevelComponent.js [hunk 1]
```diff
@@ -0,0 +1,346 @@
...
+  textCenter: {
+    textAlign: 'none',
+    fontSize: '1rem'
+  },
+  [theme.breakpoints.down('md')]: {
+    textCenter: {
+      textAlign: 'center'
+    },
+    searchBar: {
+      width: '66%',
```
Comment (kaizvn) [inline components/UniLevelComponent.js]: use grid class

File: components/TreeComponent.js [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+              styles={{
+                links: {
+                  fill: 'none',
+                  stroke: 'black',
+                  strokeWidth: '4px'
+                },
+                nodes: {
+                  node: {
+                    circle: {
+                      stroke: '#399',
```
Comment (kaizvn) [inline components/TreeComponent.js]: please make a file to reference those color hex.

**Comment by kaizvn** [comment]
remember to fix those issue above in another issues.

**Pull request merged**